src/arch/helperadvsimd.h

```diff
@@ -130,11 +130,11 @@ static INLINE vfloat vsqrt_vf_vf(vfloat d) { return vsqrtq_f32(d); }
 
 // Multiply accumulate: z = z + x * y
 static INLINE vfloat vmla_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) {
-  return vmlaq_f32(z, x, y);
+  return vfmaq_f32(z, x, y);
```

Comment:
Oh! Strange. Could you please provide a minimal example that shows what seems to be an  inconsistent behavior? You might have found a bug in gcc.